qubesusbproxy/core3ext.py

```diff
@@ -298,7 +298,11 @@ def _sanitize(
                     if i >= len(untrusted_device_desc):
                         break
                     hex_code = untrusted_device_desc[i - 1: i + 1]
-                    c = chr(int(hex_code, base=16))
+                    try:
+                        hex_value = int(hex_code, 16)
+                        c = chr(hex_value)
```

Comment:
I’d do an explicit check for both characters being hex digits.  `int()` is rather loose.